### DISCUSSION TITLE
Is the tool Visualizer gone？

### DISCUSSION THREAD

**Discussion by laomaolaile** [eclipse-cdt/cdt#621] [Q&A] [answered]
hello.

I have used Visualizer before, but when I was using version 2023.06, I couldn't find it. 

Has the Visualizer been removed? Will it be maintained in the future? Or maybe there are other tools to replace it？

![image](https://github.com/eclipse-cdt/cdt/assets/19726057/b465d6e4-c843-4d76-a67e-daed3d593f92)

thanks.

**Comment by jld01** [comment, marked as answer]
@laomaolaile the _C/C++ Multicore Visualizer_ is available for installation as an optional feature:

![image](https://github.com/eclipse-cdt/cdt/assets/45816183/52cc106e-2402-4f9e-bbc3-e7195dc7e8ef)
